src/PhpWord/Reader/Word2007/AbstractPart.php

```diff
@@ -485,6 +489,71 @@ protected function readTabsStyle(XMLReader $xmlReader, \DOMElement $domNode)
         return $tabStyles;
     }
 
+    /**
+     * Read w:drawing.
+     *
+     * @param \PhpOffice\PhpWord\Shared\XMLReader $xmlReader
+     * @param \DOMElement $domNode
+     * @return array|null
+     */
+    protected function readDrawingImageStyles(XMLReader $xmlReader, \DOMElement $domNode)
+    {
+        if (is_null($domNode)) {
+            return null;
+        }
+
+        $styles = array();
+
+        if ($xmlReader->elementExists('wp:anchor', $domNode)) {
+            $picNode = $xmlReader->getElement('wp:anchor/a:graphic/a:graphicData/pic:pic', $domNode);
+
+            // Anchor wrapping style square
+            if ($xmlReader->elementExists('wp:anchor/wp:wrapSquare', $domNode)) {
+                $styles['wrappingStyle'] = ImageStyle::WRAPPING_STYLE_SQUARE;
```

Comment:
While testing I've found a problem with the wrapping style. As you see in sample 11. Wrapping styles are not recognized by addImage with styles.